sunpycube/cube/NDCube.py

```diff
@@ -165,7 +167,18 @@ def world_to_pixel(self, quantity_axis_list, origin=0):
         return result[::-1]
 
     def to_sunpy(self):
-        raise NotImplementedError("to_sunpy() is not Implemented.")
+        wcs_axes = list(self.wcs.wcs.ctype)
+        missing_axis = self.missing_axis
+        index_not_one = []
+        if 'TIME' in wcs_axes and len(self.dimensions.shape) is 1:
+            result = self.pixel_to_world([u.Quantity(self.data, unit=u.pix)])
```

Comment:
@abit2 An easy way to define a TimeSeries is to use a pandas DataFrame.  So here I'm just creating a pandas.DataFrame with an index column = ```dt``` (seconds in WCS reference time), and two other columns: ```"property"``` (the data values) and ```"uncertainty"``` (the uncertainty of the data values).  I do the ```try...except...``` in case the ```uncertainty``` kwarg in the ```NDCube``` is not defined.